indico/modules/rb/schemas.py

```diff
@@ -401,6 +401,10 @@ def _check_room_name_format_placeholders(self, room_name_format, **kwargs):
             # validated client-side, no i18n needed
             raise ValidationError('Missing placeholders: {}'.format(', '.join(missing)))
 
+        placeholders = get_format_placeholders(room_name_format)
+        if invalid := set(placeholders) - {'building', 'floor', 'number'}:
+            raise ValidationError(_('Invalid placeholders: {}').format(', '.join(invalid)))
```

Comment:
I'd skip the i18n here; this should never trigger due to the client-side validation so no need for translators to translate it